app/repositories/payment-repository.js

```diff
@@ -16,7 +18,43 @@ export async function set (reference, data, frn) {
 export async function updatePaymentResponse (reference, status, paymentResponse) {
   const { models } = dataModels
   return models.payment.update(
-    { status, paymentResponse },
+    { status, paymentResponse, frn: paymentResponse.frn },
     { where: { applicationReference: reference } }
   )
 }
+
+export async function getPendingPayments () {
+  const { models } = dataModels
+  return models.payment.findAll({
+    where: {
+      status: Status.ACK,
+      paymentCheckCount: {
+        [Op.lt]: 3
+      },
+      frn: {
+        [Op.ne]: null
+      }
+    }
+  })
+}
+
+export async function incrementPaymentCheckCount (claimReference) {
+  const { models } = dataModels
+  return models.payment.increment(
+    { paymentCheckCount: 1 },
+    { where: { applicationReference: claimReference } } // applicationReference is actually claimReference
```

Comment:
@aaroncarroll82 is there any appetite for changing this column in the payment table from applicationReference to claimReference?